src/drt/src/dr/FlexDR_maze.cpp

```diff
@@ -2622,6 +2623,18 @@ void FlexDRWorker::processPathSeg(frMIdx startX,
                   z,
                   i - 1 >= 0 ? &points[i - 1] : nullptr,
                   i + 2 < (int) points.size() ? &points[i + 2] : nullptr);
+  } else {
```

Comment:
You could make this a little bit cleaner if you made this else block into an else if, else. It would remove one level of nesting which is generally good